backend/tests/integration/orcid/conftest.py

```diff
@@ -25,7 +25,7 @@
 import pytest
 
 IS_VCR_ENABLED = True
-IS_VCR_EPISODE_OR_ERROR = True  # False to record new cassettes.
+IS_VCR_EPISODE_OR_ERROR = False  # False to record new cassettes.
```

Comment:
leftover?

Reply:
@ammirate updated

Reply:
@drjova There are two tests failing because of `WorkAlreadyExistsException` :(